src/extension/src/file_handlers/ExtOutputStatusHandler.py

```diff
@@ -210,12 +213,14 @@ def add_error_to_status(self, message, error_code=Constants.PatchOperationErrorC
         else:
             return
 
-    @staticmethod
-    def __ensure_error_message_restriction_compliance(full_message):
+    def __ensure_error_message_restriction_compliance(self, full_message):
         """ Removes line breaks, tabs and restricts message to a character limit """
         message_size_limit = Constants.STATUS_ERROR_MSG_SIZE_LIMIT_IN_CHARACTERS
         formatted_message = re.sub(r"\s+", " ", str(full_message))
-        return formatted_message[:message_size_limit-3] + '...' if len(formatted_message) > message_size_limit else formatted_message
+        if full_message not in self.__ignore_error_message_restriction_compliance_strings:
+            return formatted_message[:message_size_limit - 3] + '...' if len(formatted_message) > message_size_limit else formatted_message
+        else:
+            return formatted_message
```

Comment:
Ensure UT coverage.

Reply:
Removed for extension - only necessary in core.